modules/services/localization/src/main/res/values/strings.xml

```diff
@@ -41,7 +41,7 @@
     <string name="go_to_files">Go to files</string>
     <string name="in_progress">In progress</string>
     <string name="log_in">Log in</string>
-    <string name="log_in_subtitle">Start listening to your favourite podcasts</string>
+    <string name="log_in_subtitle">Start listening to your favorite podcasts</string>
```

Comment:
The default strings.xml file is US English, with a UK English translation in GlotPress.